### DISCUSSION TITLE
ToSchema field descriptions with some field-specific attributes

### DISCUSSION THREAD

**Discussion by jmhrpr** [juhaku/utoipa#483] [Q&A]
Hello,

It seems that when I use an attribute like `#[serde(rename = ...)` or `#[schema(value_type = ...)]` that the doc comments for each field are not used for the field descriptions in the generated openapi json file.

My code looks something like:

```
#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, ToSchema)]
pub struct MyStruct {
    #[serde(rename = "type")]
    /// Field description one
    pub field_one: MyEnumType,
    /// Field description two
    pub field_two: String,
    /// Field description three
    pub field_three: Option<String>,
    #[schema(value_type = Option<Any>)]
    /// Field description four
    pub field_four: Option<serde_json::Value>,
}
```

But the created openapi.json only has the description for field two and three. Am I doing something wrong? Or maybe its due to the types of the field? Thanks

**Comment by jmhrpr** [comment]
Ah, for the enum type I see that I add the description by doc commenting the enum type itself. But I'm not sure what to do for in the case of field four.

**Reply by juhaku** [reply]
Hi, 

In case of `field_four`, there is no support for it at the moment unfortunately. But adding the support should be quite easy. I'll create an issue for this topic so it can be tracked.

**Reply by juhaku** [reply]
Issue can be found here: https://github.com/juhaku/utoipa/issues/491
